leader-election-impl/src/main/java/com/palantir/paxos/PaxosProposerImpl.java

```diff
@@ -203,7 +203,7 @@ public int getQuorumSize() {
     }
 
     @Override
-    public String getUUID() {
+    public String getUuid() {
```

Comment:
I undid a number of these changes in https://github.com/palantir/atlasdb/pull/1856/commits/b1e4141b4968442694efe0a0441e32e2a341f72e - does this one actually break any external products?